node/light.go

```diff
@@ -104,6 +111,9 @@ func (ln *LightNode) OnStart() error {
 	return nil
 }
 
+// OnStop stops the light node
+//
+// TODO: should this just be Stop?
```

Comment:
same as start

Reply:
The `Start` method in `BaseService` calls the `OnStart` method of its `impl`

Reply:
```suggestion
// OnStop stops the light node
```